### DISCUSSION TITLE
12 bilardo topu sorusu

### DISCUSSION THREAD

**Discussion by sonsuzus** [sonsuzus/sonsuzus#130] [Zeka Sorusu]
Şekil olarak aynı 12 bilardo topunun birisi diğerlerinden (daha ağır veya daha hafif olmak üzere) farklılık göstermektedir.
Eş kollu terazi ile 3 tartı hakkınız var. Bu farklı bilardo topunu ve ağır mı hafif mi olduğunu bulunuz.

**Comment by Sevvalgocgun1** [comment]
Her iki tarafada her seferde ikişer top koyarsak diğerinden ağır yada hafif olanı buluruz.
